src/scenic/syntax/compiler.py

```diff
@@ -449,7 +449,7 @@ def transformPropertyDef(self, node: s.PropertyDef):
                 ),
                 ast.Lambda(
                     args=selfArg,
-                    body=self.visit(node.value),
+                    body=node.value,
```

Comment:
`node.value` is visited in line 439. Removing this `visit` to avoid visiting the same node twice as some transformations are not idempotent.